packages/odyssey-react/src/components/Field/Field.tsx

```diff
@@ -176,7 +177,7 @@ function Hint({ id, children }: PropsHint) {
 
 function FieldError({ id, children }: PropsError) {
   return (
-    <Box as="p" className={styles.error} id={`${id}-error`}>
+    <Box as="p" color="danger" fontSize="caption" id={`${id}-error`} margin="0">
```

Comment:
We're setting `margin` to `0` here, but property being replaced is `margin-block-start: $spacing-xs`. Is that intentional?

Reply:
Fixed

Reply:
+1 good catch thats the only question I have on this PR